src/BizHawk.Client.EmuHawk/AVOut/FFmpegWriterForm.cs

```diff
@@ -46,32 +46,18 @@ public static FormatPreset[] GetPresets(string customCommand)
 			{
 				return new[]
 				{
-					new FormatPreset("AVI Lossless UT Video", "Lossless UT video and uncompressed audio in an AVI container. Compatible with AVISource(), if UT Video decoder is installed. Fast, but low compression.",
+					new FormatPreset("Social Media — MP4", "Optimized for quick and easy sharing. Produces small, highly compatible and streamable files. AVC video and Opus audio in a MP4 container.",
+						"-c:a libopus -c:v libx264 -pix_fmt yuv420p -movflags +faststart -f mp4", false, "mp4"),
+					new FormatPreset("[ Custom ]", "Write your own FFmpeg command. For advanced users only.",
+						customCommand, true, "foobar"),
+					new FormatPreset("Lossless AVC — MKV", "Lossless AVC video and lossless FLAC audio in a Matroska container. High speed and compression. Compatible with AVISource() if x264vfw or FFmpeg based decoder is installed. Seeking may be unstable.",
+						"-c:a flac -c:v libx264rgb -qp 0 -pix_fmt rgb24 -f matroska", false, "mkv"),
+					new FormatPreset("Lossless Ut Video — AVI", "Lossless Ut Video and uncompressed audio in an AVI container. Fast, but low compression. Compatible with AVISource(), if Ut Video decoder is installed.",
 						"-c:a pcm_s16le -c:v utvideo -pred median -pix_fmt gbrp -f avi", false, "avi"),
-					new FormatPreset("AVI Lossless FFV1", "Lossless FFV1 video and uncompressed audio in an AVI container. Compatible with AVISource(), if ffmpeg based decoder is installed. Slow, but high compression.",
+					new FormatPreset("Lossless FFV1 — AVI", "Lossless FFV1 video and uncompressed audio in an AVI container. Slow, but high compression. Compatible with AVISource(), if FFmpeg based decoder is installed.",

```

Comment:
Since FFV1 decoders seem to break all other VFW codecs (until you fix the registry, which then breaks the FFV1 decoder), I wonder if it's time to drop this option in favor of Ut.

Reply:
Source? Also this is ffmpeg, not vfw.

Reply:
You can't use ffmpeg to encode TASes with avisynth. avisynth depends on VFW decoders. VFW decoder for ffmpeg dumps breaks other VFW codecs.